Bottom: e01049fa99be7710ec41601fd75bae161ef33b15
Top:    ce72cbdc7cbe9af2002c6fee10291583b6129586
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 15:53:13 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
